spec/spec_helper.rb

```diff
@@ -17,6 +17,17 @@
 # option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+VCR.configure do |c|
+  c.cassette_library_dir = 'spec/cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.